charms/istio-pilot/src/charm.py

```diff
@@ -244,7 +243,7 @@ def _istioctl_extra_flags(self):
                     "--set",
                     f"values.cni.cniConfDir={self.model.config['cni-conf-dir']}",
                     "--set",
-                    "values.sidecarInjectorWebhook.injectedAnnotations.traffic\.sidecar\.istio\.io/excludeOutboundIPRanges=0.0.0.0/0",  # noqa
+                    "values.sidecarInjectorWebhook.injectedAnnotations.traffic\\.sidecar\\.istio\\.io/excludeOutboundIPRanges=0.0.0.0/0",  # noqa
```

Comment:
What's with the double `\\`?

Reply:
linting issue and list locally the linting was failing for me. In Python the `\` can be put before a special character, e.g. `print("\\") -> \`.

Reply:
It is weird the linting was failing for you, this line should be skipped actually. Not a blocker, but preferably it would be in a different PR.

Reply:
I tried it again, and it was not failing, but I saw warning (failure must due something else).
```bash
lint: commands[1]> pflake8 /home/rgildein/code/canonical/istio-operators/charms/istio-pilot/src/ /home/rgildein/code/canonical/istio-operators/charms/istio-pilot/tests/ /home/rgildein/code/canonical/istio-operators/charms/istio-pilot/lib/charms/istio_pilot
<unknown>:246: SyntaxWarning: invalid escape sequence '\.'
```
Reverting this change.